Jenkinsfile

```diff
@@ -23,6 +23,7 @@ pipeline {
     stage('Validate') {
       steps {
         sh 'packer validate --var-file validate-vars.json ubuntu-18-agent.json'
+        sh 'packer validate --var-file validate-vars.json windows-2019-agent.json'
```

Comment:
Would this be a good use case for a matrix based pipeline? https://jenkins.io/blog/2019/11/22/welcome-to-the-matrix/